icon-project/rewardcalculator
# IS-933 add integration test

Need test scenario file and integration test code.
* test scenario in tests/json directory in json format
* write integration test code in tests / * _ test.go
  * load and run test scenario
  * check result if necessary
